src/LaraCart.php

```diff
@@ -557,12 +564,16 @@ public static function formatMoney($number, $locale = null, $internationalFormat
      *
      * @return string
      */
-    public function feeTotals($format = true)
+    public function feeTotals($format = true, $withTax = false)
```

Comment:
really fee totals shouldn't have a tax with it either since its already calculated in the total() with everything else.